apps/react-compiler/turbo.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "extends": ["//"],
+  "tasks": {
+    "build": {
+      "env": ["DEBUG"],
+      "outputs": [".next/**", "!.next/cache/**"],
+      "dependsOn": ["^build"]
+    },
+    "dev": {
+      "dependsOn": ["^build"],
```

Comment:
This is a good config that we probably want to move to the top-level `turbo.json`. And then this local `dev` task can be removed.